src/main/content/sitemap.xml

```diff
@@ -5,37 +5,30 @@
 
     <url>
         <loc>https://openliberty.io/</loc>
-        <lastmod>2020-02-10T21:04:44+00:00</lastmod>
-        <changefreq>weekly</changefreq>
+        <changefreq>monthly</changefreq>
     </url>
     <url>
-        <loc>https://openliberty.io/docs/</loc>
-        <lastmod>2019-02-12T21:04:44+00:00</lastmod>
-        <changefreq>weekly</changefreq>
+        <loc>https://openliberty.io/docs/20.0.0.8/overview.html</loc>
```

Comment:
Can we just specify https://openliberty.io/docs/ for this which gets redirected to the right location or do we have to change it every release?

Reply:
unfortunately sitemaps are not suppose to have redirects